src/Event/Productions/EventPair.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Event\Productions;
+
+class EventPair
```

Comment:
It's definitely better than `Tuple` and I can live with `EventPair`, but maybe something like `SuggestedMatch` would even be clearer? Or `SimilarEventPair` to build on the "similarities" API name?

Reply:
I like `SimilarEventPair`